processor/probabilisticsamplerprocessor/config.go

```diff
@@ -45,6 +75,14 @@ type Config struct {
 	// despite errors using priority.
 	FailClosed bool `mapstructure:"fail_closed"`
 
+	// SamplingPrecision is how many hex digits of sampling
+	// threshold will be encoded, from 1 up to 14.  Default is 4.
+	// 0 is treated as full precision.
+	SamplingPrecision int `mapstructure:"sampling_precision"`
```

Comment:
How can we make this easier for a user to understand? How do I determine if 4 is a good number for me, and how to come up with that good number? And why 14? I think all those answers are part of the spec around R/T values, but perhaps we could provide a more accessible documentation here, while still pointing to the spec?

Reply:
Hopefully in c67350d6d0.